table/table.cpp

```diff
@@ -312,4 +360,35 @@ bool Table::correct_type(Info info, const ::database::Entry &entry) {
         default:
             return false;
     }
-} // correct_type()
\ No newline at end of file
+} // correct_type()
+
+string Table::get_type(Info &info) {
```

Comment:
Not required for this PR.

Using a string type to reflect the type of a value is an error prone practice. It leaves opportunities open to incorrectly use this data type to represent the type from info.

In a separate PR, please refactor the code to use an Enum containing the discrete states you want to represent (BOOL, INT, FLOAT, STRING, etc.) instead of string throughout the repo.